PSMDB-1624-fix-user-and-group-for-kerberos-config (#998)
percona/psmdb-docs: docs/kerberos.md

## Assumptions

> The setup of the Kerberos server itself is outside the scope of this document. Please refer to the [Kerberos documentation](https://web.mit.edu/kerberos/krb5-latest/doc/admin/install_kdc.html) for the installation and configuration steps relevant to your operating system.

Reviewer: ```suggestion
The setup of the Kerberos server itself is not included in this document. Please refer to the [Kerberos documentation](https://web.mit.edu/kerberos/krb5-latest/doc/admin/install_kdc.html) for the installation and configuration steps relevant to your operating system.
```

We assume that you have successfully completed the following steps:

* Installed and configured the Kerberos server

* Added necessary [realms](https://web.mit.edu/kerberos/krb5-1.12/doc/admin/realm_config.html)

* Added service, admin, and user [principals](https://web.mit.edu/kerberos/krb5-1.5/krb5-1.5.4/doc/krb5-user/What-is-a-Kerberos-Principal_003f.html#What-is-a-Kerberos-Principal_003f)

* Configured the `A` and `PTR` DNS records for every host running `mongod` instance to resolve the hostnames onto Kerberos realm.

## Add user principals to Percona Server for MongoDB

> To authenticate, users must exist in the Kerberos and Percona Server for MongoDB servers with names that match exactly.

Reviewer: ```suggestion
To authenticate, users must exist in the Kerberos and Percona Server for MongoDB servers. Their usernames must match exactly.
```
